### PR TITLE
Add Config type

### PR DESCRIPTION
**Pull request by danieldk**
Now with working CI builds :).

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+use std::fs::File;
+use std::io::BufReader;
+use std::ops::Deref;
+
+use pyo3::exceptions;
+use pyo3::prelude::*;
+
+use sticker_utils::{Config, TomlRead};
+
+/// Tagger configuration.
```
Comment (twuebi) [inline src/config.rs]:
~~~rust
/// Config(path)
/// --
///
/// Tagger configuration.
~~~

gives a signature via the `inspect` module:

~~~python
In [3]: inspect.signature(Config)                                                                                                                             
Out[3]: <Signature (path)>
~~~

or via IPython: 
~~~python
In [3]: sticker.Config?                                                                                                                                               
Init signature: Config(path)
Docstring:      Tagger configuration.
Type:           type
Subclasses:     
~~~

**Comment by danieldk** [comment]
> Should config be an immutable, opaque type? We could provide `__str__` and `__repr__` to allow some introspection.

I agree that it would be good to have `__str__` and `__repr__`, plus properties to change fields. But I'd like to defer the implementation until `Labeler`, `Embeddings`, and `Model` classes are implemented, since the implementation of these methods should largely be delegated to these classes. Also, the properties should be implemented on these classes.

I planned to implement these configuration classes *after* the `Tagger` class is done. First provide enough functionality to actually tag sentences, then iterate/refine ;).

File: src/config.rs (hunk 1 above)
Comment (danieldk) [reply src/config.rs]: Nice find! So it works when you put it on the class rather than the constructor.

Comment (danieldk) [reply src/config.rs]: Added, though I called it `Config(file)`, following the signature of `open`.

**Comment by twuebi** [comment]
> I planned to implement these configuration classes _after_ the `Tagger` class is done. First provide enough functionality to actually tag sentences, then iterate/refine ;).

Ok, then nothing to nag

File: src/config.rs (hunk 1 above)
Comment (danieldk) [reply src/config.rs]: But at least `help(Config)` does show the signature on top. Having a signature on `__new__` would be nicer of course.

Comment (twuebi) [reply src/config.rs]: For me, it only displays the super type:

~~~python
In [1]: from sticker import Config                                                                                                                            

In [2]: help(Config)         

Help on class Config in module builtins:

class Config(object)
 |  Tagger configuration.
 |  
 |  Methods defined here:
 |  
 |  __new__(*args, **kwargs) from builtins.type
 |      Create and return a new object.  See help(type) for accurate signature.
~~~

Comment (danieldk) [reply src/config.rs]: Interesting, this is what I get:

```
>>> help(sticker.Config)
Help on class Config in module builtins:

class Config(object)
 |  Config(file)
 |  
 |  Tagger configuration.
 |  
 |  Static methods defined here:
 |  
 |  __new__(*args, **kwargs) from builtins.type
```

```
% python --version
Python 3.7.4
```

Comment (danieldk) [reply src/config.rs]: Also works in iPython (which you seem to use?).

Comment (twuebi) [reply src/config.rs]: Am on 3.6.5, seems it was added after that release:

https://github.com/python/cpython/commit/ccb5f3cee9ebb5f20e33f31432a46579f8e9bf8e#diff-cbbbf1954ca2586f5d1d6241674fab9a

Comment (danieldk) [reply src/config.rs]: That's a nice change then!

**Pull request merged**